pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -823,6 +820,13 @@ var _ = Describe("Pod Network", func() {
 	})
 })
 
+func getIpVersionNum(proto iptables.Protocol) string {
+	if proto == iptables.ProtocolIPv6 {
+		return "6"
+	}
+	return "4"
+}
+
```

Comment:
Can we this helper return the full nftables table name ? so it's resued at both tests
```golang
func getNftablesNatName(proto iptables.Protocol) string {
        version := "4"
	if proto == iptables.ProtocolIPv6 {
		version = "6"
	}
	return fmt.Sprintf("ipv%s-nat", version)
}
```

Reply:
done